docs/platform/howto/static-ip-addresses.rst

```diff
@@ -0,0 +1,108 @@
+Static IP addresses
+===================
+
+**Note: Each static IP addresses has an hourly price ranging from $0.025
+to $0.075 depending on region. The feature is recent and
+Aiven's**\ `pricing page <https://aiven.io/pricing>`__\ **doesn't
+include static IPs yet.**
+
+By default an Aiven service will use public IP addresses allocated from
+the cloud provider's shared pool of addresses for the cloud region. As a
+result, Aiven service nodes effectively get a random IP address. Because
+the addresses are shared with all other users of the cloud region, it's
+not feasible to create firewall rules to limit access to/from nodes
+using public IP addresses as the firewall would need to grant access to
+the cloud region's entire IP range granting access to all other users of
+that cloud region.
+
+You can however create IP addresses to your project. These addresses
+have a static IP for the lifetime of the address resource, and makes the
+addresses predictable for firewall rules. After creating the address it
+can be associated with an Aiven service. Once enough IP addresses have
```

Comment:
How many IP addresses do I need to associate before turning on the Static IP flag on the service?

Reply:
It was a LOT more complicated but I figured it out! Explanation of how many IPs are needed added to the doc, and wording made clearer.